rest_api/rest_api/utils.py

```diff
@@ -22,7 +22,13 @@ def get_app() -> FastAPI:
 
     from rest_api.config import ROOT_PATH
 
-    app = FastAPI(title="Haystack REST API", debug=True, version=haystack_version, root_path=ROOT_PATH)
+    app = FastAPI(
+        title="Haystack REST API",
+        debug=True,
+        version=haystack_version,
+        root_path=ROOT_PATH,
+        servers=[{"url": "http://localhost:8000"}],
```

Comment:
I don't think we need this because we call `get_openapi` directly. To avoid any unexpected side effect I'd leave the call to `FastAPI` untouched and hardcode `servers=[{"url": "http://localhost:8000"}]` into the call to `get_openapi` below.